lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -492,23 +477,6 @@ type PostExternalTransaction = "proxy"
     :> ReqBody '[OctetStream] PostExternalTransactionData
     :> PostAccepted '[JSON] ApiTxId
 
-{-------------------------------------------------------------------------------
-                                   Internals
--------------------------------------------------------------------------------}
-
--- | Any media type
-data Any
-
-instance Accept Any where
-    contentTypes _ = ("*" // "*") :|
-        -- We also 'conveniently' accept JSON format
-        [ "application" // "json"
-        , "application" // "json" /: ("charset", "utf-8")
-        ]
```

Comment:
Not needed anymore, `servant-0.17` now has a `NoContentVerb` which doesn't require this awkward work-around for responses with `NoContent`.